src/main/java/mate/academy/bookstore/validation/Isbn.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.bookstore.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = IsbnValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Isbn {
+    String message() default "Invalid format isbn";
+    Class<?>[] groups() default {};
+    Class<? extends Payload>[] payload() default {};
+}
```

Comment:
Please check is annotation that validates ISBN already exists, to not implement logic that already implemented